### DISCUSSION TITLE
A question about terminals and dependencies

### DISCUSSION THREAD

**Discussion by GavinHuttley** [datalayer/support#5] [General]
Hi,

1. Is it possible to get a Terminal interface for teaching students about the command line?
2. Is it possible to install third-party dependencies for a class?

thanks!

Gavin

**Comment by echarles** [comment]
Hi @GavinHuttley Thank you so much for asking and very sorry for the late answer.

Third party dependencies is planned for February 2024 based on the work being done for Datalayer Run https://datalayer.run. You will be able to launch kernels with custom environments.

We plan after to add the terminal that will connect to those remote kernels.

Hope this fits your planning. Let's keep this open to track progress.
